Sources/ApolloTestSupport/XCTAssertHelpers.swift

```diff
@@ -42,6 +42,10 @@ public func XCTAssertMatch<Pattern: Matchable>(_ valueExpression: @autoclosure (
   XCTFail(message(), file: file, line: line)
 }
 
+public func XCTAssertThrowsError<T>(_ expression: @autoclosure () throws -> T, _ message: @autoclosure () -> String = "", file: StaticString = #filePath, line: UInt = #line, _ errorHandler: (Error) throws -> Void = { _ in }) rethrows {
+  
+}
```

Comment:
Not with this signature. I was trying to create an overloaded version that allows for a throwing `errorHandler` so you can use `XCTUnwrap` and `XCTDowncast` in there. Right now, you need to wrap your handler in `whileRecordingErrors` and I wanted to avoid that. But it seems I got distracted and didn't finish this :)

Reply:
Why would you want to use those on an error - to check associated values?

Reply:
Yes, you want to cast `Error` to `GraphQLError` for example, and test the properties of the error.

Reply:
Got it. Long term, I think we may want to consider renaming these custom test methods to not use `XCT` since they're not apple-provided, but I'm fine with punting on that to another PR.